App/SB/components/ThreadDetailCard/index.js

```diff
@@ -60,7 +60,11 @@ const ThreadDetailCard = props => {
       }
 
       const defaultSource = require('../../views/Settings/statics/main-image.png')
-      const uri = props.photo.author_id ? 'https://cafe.us-east-1.textile.io/ipns/' + props.photo.author_id + '/avatar' : undefined
+      let uri = props.photo.author_id ? 'https://cafe.us-east-1.textile.io/ipns/' + props.photo.author_id + '/avatar' : undefined
+      // ensure we have the user's latest avatar even if the cafe is still caching
+      if (props.profile && props.profile.id && props.profile.id === props.photo.author_id) {
+        uri = 'https://cafe.us-east-1.textile.io' + props.profile.avatar_id
```

Comment:
I thought `props.profile.avatar_id` is actually a full URL. It's confusing, we should update that name.

Reply:
just reflects the key in go, @sanderpick is there a tie to avatar_id?

Reply:
I'm just saying that I think your concat of `https://cafe.us-east-1.textile.io` with `props.profile.avatar_id` will result in ``https://cafe.us-east-1.textile.iohttps://cafe.us-east-1.textile.io/someId`

Reply:
avatar_id looks something like, `/ipfs/QmWUnMBtSRzaMTj622aHESaRtHLdbu93XvxSMug31VvGkT/thumb?key=18lJGIOSnCS7xA3uBAFEuAEdEHD18lJGOWJfbsZl5PuA`

Reply:
Ah ok, that new. Didn't know.